src/backend/marsha/core/lti/utils.py

```diff
@@ -9,6 +9,57 @@ class PortabilityError(Exception):
     """An error raised when trying to access a resource that is not portable."""
 
 
+def get_resources(model, lti):
```

Comment:
This function doesn't return `resources` but a `querySet`, you should rename this function